docs/decision_record.md

```diff
@@ -9,6 +9,32 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 [TOC]
 
 
+## 2023-06-28 - SeqCol JSONschema defines reserved attributes without additional namespacing
+
+### Decision
+
+One potential issue is the possibility for a future "official" attribute to clash with a custom attribute implemented by a third party. If a custom implementation uses an attribute that a future version of seqcol will add to the "official" schema, and if these attributes are defined differently, then the custom implementation wouldn't be compatible with the future seqcol schema. It would be nice if we could prevent such future clashes by keeping key words meaning the same thing across collections so that they are compatible. One way to solve this would be to define an "official" namespace, so that custom attributes look different from official attributes. This would guarantee that a custom attribute never clashes with an official attribute, thereby ensuring that custom implementations will be compatible with any future official schema updates.
+
+Despite the potential issue for custom attribute clashes, we decided:
+
+1. We will not use any additional namespacing. Instead, the SeqCol schema declares and defines the specific attributes of a sequence collection. We will "claim" any reserved keywords in the "official" schema we publish, not by defining a style or namespace of reserved keywords.
+
+2. We will try to add to this many things that we forsee as possible attributes that could be defined in a seqcol. Thus, we will provide an official set of definitions that should prevent many possible future clashes.
+
+3. We will specify that for custom attributes, you can do what you want outside the reserved keywords; but you should be aware that if a word becomes part of the official schema in the future, this could require a change of your custom attribute to maintain backwards compatibility. We will advise that if possibility of future clashes is important for an external schema, they could prevent that by prefixing custom attributes. However, this also means that if a future attribute *is* added to the schema to represent that concept, it would not follow the custom name.
```

Comment:
Should we also invite implementation to propose custom attribute names and definition and maintain a list separate from the specification but still publicly available?
This could be a set of issues in GitHub for example.

Reply:
How about this?

> 4. Third-party implementers may propose attributes that should be moved into the primary seqcol schema for subsequent release. These proposals could happen via raising an issue in the specification repository.